<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="drop_widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>771</width>
      <height>671</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>651</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(20, 33, 61);
background-size: cover;
background-position: top;
border-top-left-radius:50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>-10</y>
       <width>431</width>
       <height>651</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255);
border-bottom-right-radius:50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Configs</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="location_input">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>160</y>
       <width>255</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sendTo_input">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>250</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Phone or Email</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>420</y>
       <width>255</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 8px;
background:#14213d;
height: 30px;
color: #fff;

}
QPushButton:hover{
background-color:rgb(0,0,0);
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}

</string>
     </property>
     <property name="text">
      <string>Monitor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#fff;</string>
     </property>
     <property name="text">
      <string>Kindly, fill the form on the right to continue</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cls_button">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>0</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{

border-radius: 13.5px;
background:#14213d;
color: #fff;

}
QPushButton:hover{
background-color:#e63946;
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}

</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>620</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#14213d;</string>
     </property>
     <property name="text">
      <string>Abwape.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="min_butto">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>0</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{

border-radius: 13.5px;
background:#e5e5e5;
color: #000;

}
QPushButton:hover{
background-color:#06d6a0;
color:#fff;
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}

</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file_name">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>300</y>
       <width>251</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Browse">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>370</y>
       <width>255</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 8px;
background:#14213d;
height: 30px;
color: #fff;

}
QPushButton:hover{
background-color:rgb(0,0,0);
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}

</string>
     </property>
     <property name="text">
      <string>Upload Video</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
